modules/widget-visibility/widget-conditions.php

```diff
@@ -220,6 +251,7 @@ public static function widget_conditions_admin( $widget, $return, $instance ) {
 									<option value="tag" <?php selected( "tag", $rule['major'] ); ?>><?php echo esc_html_x( 'Tag', 'Noun, as in: "This post has one tag."', 'jetpack' ); ?></option>
 									<option value="date" <?php selected( "date", $rule['major'] ); ?>><?php echo esc_html_x( 'Date', 'Noun, as in: "This page is a date archive."', 'jetpack' ); ?></option>
 									<option value="page" <?php selected( "page", $rule['major'] ); ?>><?php echo esc_html_x( 'Page', 'Example: The user is looking at a page, not a post.', 'jetpack' ); ?></option>
+									<option value="post_type" <?php selected( "post_type", $rule['major'] ); ?>><?php echo esc_html_x( 'Post Type', 'Show/Hide if user in certain custom post type archive.', 'jetpack' ); ?></option>
```

Comment:
This string "Show/Hide if user in certain custom post type archive." should be "Show/Hide if user is in a certain custom post type archive."